DisguiseUnityRenderStream/Runtime/DisguisePresenter.cs

```diff
@@ -0,0 +1,233 @@
+
+        /// <summary>
+        /// The index of the selection in the texture dropdown to present to the screen.
+        /// The dropdown choices are generated inside <see cref="GetManagedRemoteParameters"/>, as a concatenated list of:
+        /// None + Channels (output) + Live textures (input).
+        /// </summary>
+        public int Selected
+        {
+            get => m_Selected;
+            set => m_Selected = value;
+        }
+        
+        /// <summary>
+        /// The <see cref="PresenterResizeStrategies">strategy</see> for resizing the selected texture to screen.
+        /// </summary>
+        public PresenterResizeStrategies ResizeStrategy
+        {
+            get => m_ResizeStrategy;
+            set => m_ResizeStrategy = value;
+        }
+        
+        [SerializeField]
+        int m_Selected;
+
+        [SerializeField]
+        PresenterResizeStrategies m_ResizeStrategy = PresenterResizeStrategies.Fit;
+
+        [SerializeField]
+        CameraCapturePresenter m_OutputPresenter;
+        
+        [SerializeField]
+        Presenter m_InputPresenter;
+
+        CameraCapture[] m_Outputs = {};
+        RenderTexture[] m_Inputs = {};
+
+        static BlitStrategy.Strategy PresenterStrategyToBlitStrategy(PresenterResizeStrategies strategy) => strategy switch
+        {
+            PresenterResizeStrategies.ActualSize => BlitStrategy.Strategy.NoResize,
+            PresenterResizeStrategies.Stretch => BlitStrategy.Strategy.Stretch,
+            PresenterResizeStrategies.Fill => BlitStrategy.Strategy.Fill,
+            PresenterResizeStrategies.Fit => BlitStrategy.Strategy.Letterbox,
+            PresenterResizeStrategies.Clamp => BlitStrategy.Strategy.Clamp,
+            _ => throw new ArgumentOutOfRangeException()
+        };
+
+        /// <summary>
+        /// Instantiates a prefab with a GameObject hierarchy configured to be dropped into a scene.
+        /// It contains the <see cref="DisguisePresenter"/> and <see cref="DisguiseRemoteParameters"/> components,
+        /// as well as the necessary <see cref="m_OutputPresenter"/> and <see cref="m_InputPresenter"/>.
+        /// </summary>
+        /// <returns></returns>
+        public static GameObject LoadPrefab()
+        {
+            return Resources.Load<GameObject>(k_PrefabPath);
+        }
+        
+#if UNITY_EDITOR
+        /// <summary>
+        /// Returns the list of remote parameters to control the presenter.
+        /// The parameters are pre-configured in the prefab used by <see cref="LoadPrefab"/>.
+        /// </summary>
+        /// <remarks>
+        /// The choices for the texture selection dropdown are scene-specific and correspond to a concatenated list of:
+        /// None + Channels (output) + Live textures (input).
+        /// </remarks>
+        public static List<ManagedRemoteParameter> GetManagedRemoteParameters(ManagedSchema schema, ManagedRemoteParameters sceneSchema)
```

Comment:
Curiosity: Why do we have this special mechanism for those remote parameters?  Can't they be simply picked-up as normal remote parameters?

Reply:
I wanted to add the presenter to all scenes automatically. One method would be to add it to each scene in a build postprocessor but I'm not a fan of modifying scene files under the hood.

This other method is to add it to each scene at runtime after it's loaded. We only need to make sure it's in the schema by retrieving the presenter parameters from `GetManagedRemoteParameters`.

It also does some processing on the parameters, most importantly we need to collect the texture names for the dropdown (channels are global, but live textures are per-scene). So `GetManagedRemoteParameters` needs to be called after a first version of the schema has been generated.